src/backend/_scss/application.scss

```diff
@@ -2175,3 +2175,61 @@ a.skip:hover {
   display: block;
 }
 
+.dropdown-submenu a h4 span {
+  display: none;
+}
+
+.dropdown-inner {
+  max-height: 200px;
+  overflow: scroll;
+}
+
+@media only screen and (max-width:900px) {
+  .filter {
+    margin-right: 1%;
+  }
+}
+
+@media only screen and (max-width:989px) {
+  .sidebar{
+    display: none;
+  }
+
+  .tracks-rooms-filter {
+    margin-right: 10px;
+    padding-top: 10px;
+  }
+
+  .search-filter-schedule {
+    width: 39%;
+  }
+
+  .dropdown-menu>li {
+    cursor: pointer;
+    position: relative;
+    text-align: left;
+  }
+
+  .dropdown-menu>li>a {
+    padding-left: 5px;
+  }
+
+  .dropdown-menu>li>a>h4 {
+    text-align: left;
+  }
+
+  .dropdown-submenu {
+    position: static;
+  }
```

Comment:
Some track/room names can be quite long, too wide to fit in the dropdown, causing them to wrap.

But now that the dropdown is opening below, I think we can make it a bit wider. (What is the smallest mobile device width? 480px?)

The other tweaks facilitate wrapping of the room/track name, when it does still happen. Hopefully the comments explain what they are for.

You are welcome to use these rules if you like, and adjust them as you see fit.

```sass
  .dropdown-inner {
    // Some track/room names can be quite long.  Expand the submenu for them.
    min-width: 300px;

    > li {
      // If the track/room name needs to wrap, it should wrap in its own box, not below the coloured dot
      display: flex;
      flex-direction: row;
      align-items: center;

      // Override the line-height we have inherited from .date-list
      line-height: 1.5em;
      // But allow some spacing between entries
      padding: 2px 0px;

      .titlecolor {
        // Do not let the dot lose its shape from being shrunk
        flex: 0 0 auto;
        // Ensure a gap between the dot and the track/room name
        margin-right: 8px;
      }
    }
  }
```

We could increase the font-size just a little, and remove the padding.